pkg/solana/chainwriter/ccip_example_config.go

```diff
@@ -42,7 +39,7 @@ func TestConfig() {
 					Accounts: PDALookups{
 						Name: "RegistryTokenState",
 						PublicKey: AccountConstant{
-							Address:    registryAddress,
+							Address:    routerProgramAddress,
 							IsSigner:   false,
 							IsWritable: false,
```

Comment:
I think most of the instances were cleaned up in this PR but noticed there were still a few cases where we're setting these fields in `PDALookups`'s `PublicKey`:
- RegistryTokenConfig
- CPISigner
- RouterAccountConfig
- RouterAccountState